tomls/omnibus-mainnet/governance/councils-satellite.toml

```diff
@@ -1,7 +1,7 @@
 [clone.governance_satellite_ambassador]
 source = "<%= settings.governance_satellite_package %>"
 targetPreset = "ambassador"
-options.salt = "<%= settings.salt %>-ambassador"
+options.salt = "<%= settings.salt %>-ambassador-2"
```

Comment:
won't it redeploy just because of `governance_administration_period_duration` change anyway? Seems unnecessary

Reply:
i havent checked for sure but the `governance_administration_period_duration` is set on the `initOrUpgradeElectionModule` call, so it wont trigger a new proxy deployment

Reply:
it can't reinitialize the proxy with this setting

Reply:
it can't reinitialize the proxy with this setting like db said

Reply:
Yes, it is needed and is the idea, because we need to update the settings, but it should only trigger a re-deploy on the proxy and not the all the contracts.